collectors/diskspace.plugin/metadata.yaml

```diff
@@ -88,8 +88,24 @@ modules:
         examples:
           folding:
             enabled: true
-            title: ""
-          list: []
+            title: Config
+          list: |
+            - name: Default configuration
+              description: When plugin configuration is not modifyed, plugin will collect data according the following pattern: It will not keep unmounted disks looking for mount point status every 15 seconds, but it update charts very second. Filesystems and paths listed in exclude options won't be collected, and data for inode and disk space will be collected when it is possible.
```

Comment:
```suggestion
              description: When plugin configuration is not modified, plugin will collect data according the following pattern: It will not keep unmounted disks looking for mount point status every 15 seconds, but it update charts very second. Filesystems and paths listed in exclude options won't be collected, and data for inode and disk space will be collected when it is possible.
```